integration/experiment/report.py

```diff
@@ -14,7 +14,9 @@
 def extract_trial(name):
     ''' Remove the iteration number after the last underscore. '''
     pieces = name.split('_')
-    return '_'.join(pieces[:-1]), int(pieces[-1])
+    # Assuming following report name template: 
+    # <appname>_boardcap_<platform powercap val>_core_<freq val>_uncore_<freq val>_trial_<val>_<sweep-name>-<hostname>.report
```

Comment:
```suggestion
    # Assuming report file name ends in:
    # ...trial_<TRIAL_NUM>_<SWEEP_NAME>-<HOSTNAME>.report
    # e.g. nekbone_boardcap_3000_core_2000_uncore_1000_trial_12_powersweep-node1.report
```

Use all caps inside <> to denote a variable value in the name.  Update the example based on your usages and expected values.